extension/src/main/java/com/cliqz/extension/CliqzAPI.kt

```diff
@@ -0,0 +1,107 @@
+    val HUMANWEB_MODULE = "human-web-lite"
+    val INSIGHTS_MODULE = "insights"
+
+    /**
+     * Gets the current enabled status of Cliqz modules running in the extension.
+     */
+    suspend fun getModuleStatus(): ModuleStatus? {
+        val moduleStatus = extension.callActionAsync(scope, "core", "status").await().let { (it as JSONObject).getJSONObject("modules") }
+
+        fun isModuleEnabled(module: String) = moduleStatus.has(module) && moduleStatus.getJSONObject(module).optBoolean("isEnabled", false)
+
+        return ModuleStatus(
+                isModuleEnabled(ADBLOCKER_MODULE),
+                isModuleEnabled(ANOLYSIS_MODULE),
+                isModuleEnabled(ANTITRACKING_MODULE),
+                isModuleEnabled(AUTOCONSENT_MODULE),
+                isModuleEnabled(COOKIEMONSTER_MODULE),
+                isModuleEnabled(HUMANWEB_MODULE),
+                isModuleEnabled(INSIGHTS_MODULE)
+        )
+    }
+
+    /**
+     * Set the enabled status of an extension module.
+     */
+    suspend fun setModuleEnabled(module: String, enabled: Boolean) {
+        extension.callActionAsync(scope, "core", if (enabled) "enableModule" else "disableModule", module).await()
+    }
+
+    /**
+     * Get aggregated privacy stats over the given {StatsPeriod}.
+     */
+    suspend fun getBlockingStats(period: StatsPeriod): BlockingStats? {
+        return extension.callActionAsync(scope, "insights", "getDashboardStats", period.period).await().let { (it as JSONObject) }.let { res ->
+            BlockingStats(
+                    res.optInt("adsBlocked", 0),
+                    res.optInt("cookiesBlocked", 0),
+                    res.optInt("dataSaved", 0),
+                    res.optString("day", ""),
+                    res.optInt("fingerprintsRemoved", 0),
+                    res.optInt("loadTime", 0),
+                    res.optInt("pages", 0),
+                    res.optInt("timeSaved", 0),
+                    res.optJSONArray("trackers").let {
+                        if (it != null) List(it.length()) { i -> it.getString(i) } else emptyList()
+                    },
+                    res.optJSONArray("trackersDetailed").let {
+                        if (it != null) List(it.length()) { i ->
+                            val tracker = it.getJSONObject(i)
+                            TrackerInfo(tracker.getString("name"), tracker.getString("cat"), tracker.getString("wtm"))
+                        } else emptyList()
+                    },
+                    res.optInt("trackersDetected", 0)
+            )
+        }
+    }
+}
+
+data class ModuleStatus(
+        val adblockerEnabled: Boolean,
+        val anolysisEnabled: Boolean,
+        val antitrackingEnabled: Boolean,
+        val autoconsentEnabled: Boolean,
+        val cookieMonsterEnabled: Boolean,
+        val humanwebEnabled: Boolean,
+        val insightsEnabled: Boolean
+)
+
+enum class StatsPeriod(val period: String) {
+    ALL_TIME(""),
```

Comment:
Indentation in this file is not very consistant

Reply:
I'm not sure about formatting. I ran the default formatter on all these files and it came out with this. It's passing the linter, so I guess this is how it should be? @spacifici ?

Reply:
hi, @sammacbeth It isn't running the linter checks. Since you changed the name of the 'privacy' module, you need to add it to the detekt/ktlint task in daisy/build.gradle.

Also, the project follows 4 spaces of indentation, probably this won't be detected in the linters.

PR looks good otherwise 👍